lib/dialects/postgres/connection-manager.js

```diff
@@ -119,6 +119,7 @@ ConnectionManager.prototype.disconnect = function(connection) {
   return new Promise(function (resolve, reject) {
     connection.end();
     resolve();
+    return null;
```

Comment:
I have no idea - I'm getting this warning for this line in my CI tests, this was my attempt to remove the warning (didn't work).